### DISCUSSION TITLE
Unable to See Google Pay Option in Payment Sheet Using flutter_stripe Package

### DISCUSSION THREAD

**Discussion by Nikunj-Flutter** [flutter-stripe/flutter_stripe#1895] [Q&A]
Hi everyone,

I'm currently working on a Flutter project where I'm integrating Stripe payments using the flutter_stripe package. I've followed the documentation for setting up Google Pay, but I'm unable to see the Google Pay option in the payment sheet.

```
await Stripe.instance.initPaymentSheet(
        paymentSheetParameters: SetupPaymentSheetParameters(
          primaryButtonLabel: "Pay",
          customFlow: false,
          merchantDisplayName: StringResources.appName,
          setupIntentClientSecret: clientSecret,
          customerEphemeralKeySecret: ephemeralKey,
          applePay: const PaymentSheetApplePay(
            merchantCountryCode: 'CA',
          ),
          googlePay: const PaymentSheetGooglePay(
            merchantCountryCode: 'CA',
            testEnv: true,
          ),
          style: ThemeMode.system,
        ),
      );
```

Here’s what I've done so far:

> Integrated flutter_stripe package and configured it according to the official documentation.
> Ensured that my PaymentConfiguration includes the necessary Google Pay parameters.
> Tested with different devices and environments (emulator/physical device).
> Despite this, the Google Pay option doesn’t appear when the payment sheet is displayed. Instead, I only see the card payment option.

Here are some details about my setup:

> Flutter version: 3.22.3
> flutter_stripe version: 11.0.0
> Device/Emulator: Redmi note 7, Pixel 7, Pixel 8

Has anyone faced a similar issue or can provide guidance on what I might be missing? Any help would be greatly appreciated!

Thanks in advance!

**Comment by remonh87** [comment]
seems like some thing is broken since latest upgrade we are investigating it currently. See issue: https://github.com/flutter-stripe/flutter_stripe/issues/1861

**Reply by Nikunj-Flutter** [reply]
@remonh87 Could you please suggest an alternative or a temporary workaround for this issue?

**Reply by remonh87** [reply]
for google pay I would recommend using the [pay](https://pub.dev/packages/pay) plugin for now
